src/AzureAuth/Commands/Ado/CommandPat.cs

```diff
@@ -23,9 +35,48 @@ public class CommandPat
         private const string ScopeOption = "--scope";
         private const string ScopeHelp = "A token scope for accessing Azure DevOps resources. Repeated invocations allowed.";
 
+        private const string PromptHintOption = "--prompt-hint";
+        private const string PromptHintHelp = "A prompt hint to contextualize prompts and identify uses in telemetry, when captured.";

```

Comment:
These are already defined as `public const` in CommandAad for re-use by other commands needing to do auth. Same for AuthMode, and Domain. Happy to see the content improvements made in those shared consts :)

Reply:
Done. I made some improvements to the shared consts as well.